LondonBritishLibrary/orient/BLorient8819.xml

```diff
@@ -0,0 +1,278 @@
+                <title>Discourses on the Passion for Holy Week, Salām to Jesus Christ, Salām to the Virgin Mary</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 8819</idno>
+                        <altIdentifier><idno>Or. 8819</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 57</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="4r" to="78r"/> 
+                            <title ref="LIT6193HomiliaryPassions">Discourses on the Passion of our Lord Jesus Christ for Holy Week</title>  
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="4r"/> 
+                                <title ref="LIT6193HomiliaryPassions#Introduction"/>
+                                <note>Rhyming</note>
+                                <incipit xml:lang="gez">በስመ፡ እግዚአብሔር፡ ሥሉስ፡ ዘእንበለ፡ ቀዲም፡ ወተድኅሮ። በግጻዌሁ፡ ይትረአይ፡ ዘአስተዋሐደ፡ ምክሮ። በጠፈረ፡ በረድ፡ ወነድ፡
+                                    ዘሐነጸ፡ ማኅደሮ። ወበአርእስተ፡ ኪሩቤል፡ ርቱዓ፡ ዘአማዕዘነ፡ መንበሮ። </incipit>
+                            </msItem>
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="6v"/> 
+                                <title ref="LIT6193HomiliaryPassions#Monday"/>
+                                <note>Indicated by <foreign xml:lang="gez">ዘሰኑይ፡ </foreign> on the upper margin of <locus target="#4r"/>.</note>
+                                <incipit xml:lang="gez">ናንቅሕ፡ ርእሰ፡ ሕሊናነ፡ ወናስተሐውስ፡ አስራወ፡ ልሳንነ፡ ንክሥት፡ አፉነ፡ ወናስተናብብ፡ ከናፍሪነ። ከመ፡ ንንግር፡ ዜና፡ ሕማማቲሁ፡ 
+                                    ለመድኅኒነ፡ እንዘ፡ ኃያል፡ ዘኢየሐምም፡</incipit>
+                            </msItem>
+                            <msItem xml:id="ms_i1.3">
+                                <locus from="17r"/> 
+                                <title ref="LIT0000000000000000000">Homily for Tuesday</title>
```

Comment:
Maybe provisionally an additional div can be created under LIT6193HomiliaryPassions as "TuesdayOld" or something, with a note there that it is rarely attested (?)